### PR TITLE
Readme

### LINKED ISSUE #11: Read me file
Creat a read me file

### PR DESCRIPTION
**Pull request by razvanbrb**
Please check read me file. Issue #11 . Thanks

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Flex Panels Image Gallery
+
+This is a simple static website which enables user to flex a gallery of images when click on it. 
+
+You can check it out live page from this [link](abelroland.github.io/flex-panel/).
+
+This project was created from the tutorials of [Wes Bos](https://github.com/wesbos/JavaScript30/tree/master/01%20-%20JavaScript%20Drum%20Kit) . We recommend you to watch his tutorials.
```
Comment (abelRoland) [inline README.md]: Hey @razvanbrb , the link here is sending to the challenge N. 1 (Drum Kit). Maybe we should send the readers, or for the master folder of the project, or for the folder of the `flex panel` challenge 😉

**Review by abelRoland — CHANGES_REQUESTED** [review]
I gave just a comment on the `readme.md` file. The rest seems very good. Thanks for the work!

**Review by yildiraykoyuncu — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (razvanbrb) [reply README.md]: Ok, good that you notice. I will change it. :+1:

**Comment by Feruzteame** [comment]
It looks good but @razvanbrb may be if it is possible can you put the screenshot of live.

**Review by abelRoland — APPROVED** [review]
👍 Perfect

**Review by Feruzteame — APPROVED** [review]

**Pull request merged**